docs/src/main/sphinx/connector/mysql.md

```diff
@@ -276,6 +276,22 @@ this table:
 
 No other types are supported.
 
+### Timestamp type handling
+
+MySQL `TIMESTAMP` types with are mapped to a Trino `TIMESTAMP WITH TIME ZONE`.
+To preserve time instants, Trino sets the session time zone
+of the MySQL connection to match the JVM time zone.
+Because of this, users may experience an issue if the JVM time
+zone does not exist on the MySQL server, receiving an error similar to:
+
+```
+com.mysql.cj.exceptions.CJException: Unknown or incorrect time zone: 'UTC'
+```
+
+Users will need to use a time zone that is known to both the JVM and MySQL,
```

Comment:
Replacement paragraph.

```
To avoid the errors, you must use a time zone that is known on both systems, or [install the missing time zone on the MySQL server](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation).
```